src/main/java/com/gregtechceu/gtceu/common/machine/multiblock/steam/SteamParallelMultiblockMachine.java

```diff
@@ -44,13 +44,14 @@
 @MethodsReturnNonnullByDefault
 public class SteamParallelMultiblockMachine extends WorkableMultiblockMachine implements IDisplayUIMachine {
 
-    public static final int MAX_PARALLELS = 8;
+    public int MAX_PARALLELS = ConfigHolder.INSTANCE.machines.steamMultiParallelAmount;
```

Comment:
```suggestion
    public int maxParallels = ConfigHolder.INSTANCE.machines.steamMultiParallelAmount;
```
Make this not all caps, refactor all usages, it's not a final var anymore